tempel.el

```diff
@@ -144,6 +144,21 @@ may be named with `tempel--name' or carry an evaluatable Lisp expression
     ((or 'n 'n> '> '& '% 'o) " ")
     (_ "_")))
 
+(defun tempel--expand-paths (locations)
```

Comment:
These are only nits of course. Your PR looks good now. locations->path or paths? I'd like this a bit more consistent.